src/core/api/repository.go

```diff
@@ -39,6 +38,9 @@ import (
 	"github.com/goharbor/harbor/src/common/utils/notary"
 	"github.com/goharbor/harbor/src/common/utils/registry"
 	"github.com/goharbor/harbor/src/core/config"
+	"github.com/goharbor/harbor/src/core/notifier"
```

Comment:
I remembered we'll move it to the `pkg` package, WHY it is still there (in core)?

Reply:
notifier is related to all service package like webbook, jobservice and so on in these handlers. there are one way to decouple this, move handler files to service package, but that will make the notifier framework scattered. so we decide to leave it in core. it is also a service framework which contains all kinds of handlers.

Reply:
> please move the notifier into pkg

not process in this PR